### ISSUE TITLE
Tensorboard file writer not working any more after nightly build tf-nightly 1.14.1.dev20190410

### ISSUE THREAD

**Issue by markste-in** [tensorflow/tensorflow#27713]
I had to restart my script this morning and it automatically installs the latest nightly build of tensorflow (no other changes)

Unfortunately the file write doesn't produce any output anymore.
Yesterday with build 2.0.0-dev20190405 it worked just fine.

**System information**
- Colab with tf-nightly 1.14.1.dev20190410 and w.o. HW acceleration
- installed with !pip install tf-nightly-2.0-preview

**Describe the current behavior**
Tensorboard File Writer does not create any output files. The folder itself is still created.

**Code to reproduce the issue**
```
writer = tf.summary.create_file_writer(TENSORBOARD_LOG)               
with writer.as_default():
    tf.summary.scalar('running_reward', running_reward,current_run)
    for w in self.model.trainable_variables:
        tf.summary.histogram(name=w.name, data=w, step=current_run)

```

**Comment by nfelt** [comment]
@markste-in Thanks for the report.  I'm not able to reproduce this with `tf-nightly-2.0-preview==2.0.0.dev20190416`.

If you're still experiencing the issue, could you provide the contents of `pip freeze` in the environment where you are running the code?  I'm not totally clear from your description whether you installed `tf-nightly`, `tf-nightly-2.0-preview`, or both (and which versions).  If you're using the 2.0 preview, it's best to `pip uninstall tf-nightly` first, and also make sure you're using an up-to-date tensorboard nightly (via `pip install -U tb-nightly`).

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by markste-in** [comment]
Sorry for the late answer. It seems fixed with the latest version (2.0.0-dev20190516)

**Comment by manivaradarajan** [comment]
Closing based on https://github.com/tensorflow/tensorflow/issues/27713#issuecomment-493163593
